client/ayon_deadline/plugins/publish/global/submit_publish_job.py

```diff
@@ -227,8 +226,12 @@ def _submit_deadline_post_job(self, instance, job, instances):
             'publish',
             '"{}"'.format(rootless_metadata_path),
             "--targets", "deadline",
-            "--targets", "farm"
+            "--targets", "farm",
         ]
+        # TODO remove when AYON launcher respects environment variable
+        #   'AYON_DEFAULT_SETTINGS_VARIANT'
+        if settings_variant == "staging":
+            args.append("--use-staging")
```

Comment:
So - what if this is running as `dev`? Or is that irrelevant because the bundle itself then already defines it as `dev`? Or?

Reply:
Dev works (by accident), because bundle name points to dev bundle.